### DISCUSSION TITLE
Include URI to this repository in output of "version" and/or "man"

### DISCUSSION THREAD

**Discussion by jmtd** [ankitpokhrel/jira-cli#454] [Ideas] [answered]
There are many tools which build a binary called "jira". I downloaded the binary build for this one and the tarball only contains the binary and the LICENSE file. I wanted to find the GitHub repository that corresponded to the tool in order to do the setup steps correctly etc

But, there's no link back to, or reference to, the GitHub URI (or any other) in the output of `jira version` or `jira man`.

I suggest a link back to this repository in the output of one or the other.

(I found it by searching for the`GitCommit` hash that version printed, that won't always work :))

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @jmtd, never thought about this and I think its a wise idea. Added a link to the project in `help` and will be available in the next release.

![Screen Shot 2022-09-02 at 9 17 48 AM](https://user-images.githubusercontent.com/2364546/188080779-d67952bc-aef7-4450-b3af-5c10d131110e.png)
